src/store/modules/global/saga.ts

```
@@ -774,7 +812,38 @@ export function* createBlogSaga(action: PayloadAction<any>): Generator<any, void
     console.error('Fetch error:', error)
   }
 }
+export function* createUserBlogSaga(action: PayloadAction<any>): Generator<any, void, any> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)